src/model/modifier/__tests__/AtomicBlockUtils-test.js

```diff
@@ -36,6 +36,7 @@ const getInvariantViolation = msg => {
     /* eslint-disable fb-www/sprintf-like-args */
     invariant(false, msg);
     /* eslint-enable fb-www/sprintf-like-args */
+    return 'Should not reach here.';
```

Comment:
Is this needed?

Reply:
There'll be an ESLint error without it. `Expected to return a value at the end of arrow function. eslint(consistent-return)`

It expects the function to return something from both `try` and `catch` blocks.
```js
const getInvariantViolation = msg => {
  try {
    /* eslint-disable fb-www/sprintf-like-args */
    invariant(false, msg);
    /* eslint-enable fb-www/sprintf-like-args */
    return 'Should not reach here.';
  } catch (e) {
    return e;
  }
};
```